code/datums/status_effects/neutral.dm

```diff
@@ -348,7 +348,11 @@
 				to_chat(owner, span_userdanger("You start to convlse violently as you feel your consciousness merges across realities, your possessions flying wildy off your body!"))
 				owner.Jitter(200)
 				owner.Knockdown(10)
-			var/items = owner.get_contents()
+			var/list/items = list()
+			for (var/obj/item/item in owner.get_contents())
```

Comment:
rather than iterating the entire list, loop the pick for n (n of like max 10) tries, it's unlikely to permanently pick a no drop item from the list and it usually will be the faster path